psp/src/lib.rs

```diff
@@ -19,6 +19,8 @@
 // For the `const_generics` feature.
 #![allow(incomplete_features)]
 
+#![recursion_limit = "256"]
```

Comment:
rustc told me to add this because the macros were recursing too hard?

Reply:
Can this be removed now or is it still necessary?

Reply:
Looks good now.